src/main/java/org/semux/consensus/ActivatedForks.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2017-2018 The Semux Developers
+ *
+ * Distributed under the MIT software license, see the accompanying file
+ * LICENSE or https://opensource.org/licenses/mit-license.php
+ */
+package org.semux.consensus;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Tracks which forks are currently activated.
+ */
+public final class ActivatedForks {
```

Comment:
I don't think use a global static class object to store the fork information is a good idea. While we can benefit from saving a parameter, it would make unit test much harder and is not modularization friendly.